partitioner/src/main/java/io/confluent/connect/storage/partitioner/PartitionerConfig.java

```diff
@@ -98,6 +98,12 @@
   public static final String TIMESTAMP_FIELD_NAME_DEFAULT = "timestamp";
   public static final String TIMESTAMP_FIELD_NAME_DISPLAY = "Record Field for Timestamp Extractor";
 
+  public static final String TIMESTAMP_FIELD_SOURCE_CONFIG = "timestamp.source";
```

Comment:
This setting tells the connector where to get the timestamp field, either `key` or `value`

Reply:
Should this be a comment in the code then?

Reply:
Absolutely! 👍